ako-operator/controllers/statefulset_test.go

```diff
@@ -308,12 +308,12 @@ var sfJson = `
                         ],
                         "resources": {
                             "limits": {
-                                "cpu": "250m",
```

Comment:
@DixitAakash : I think we will require to include all newly added parameter in statefulset over here to test it.